inferno-framework/smart-app-launch-test-kit
# FI 3093: Transition to use auth info

# Summary

This branch fully transitions all versions of the SMART App Launch (STU1, STU2, and STU2.2) to use the `auth_info` feature for collecting authentication credentials during tests, replacing the use of `oauth_credentials`. 

# Testing Instructions
- Run the tests in the browser to verify the correct display of inputs and ensure appropriate test behavior.
- Execute the unit tests to confirm all tests pass successfully.

**Todo:**
- [ ] Point the `inferno_core` gem to its repository's `main` branch (required to customize the `auth_info` authentication type options) and test the `auth_type` customization.  
- [x] Update the presets to align with this implementation.  
- [ ] When a new release is available, bump the `inferno_core` version.
